types/suman/index.d.ts

```diff
@@ -0,0 +1,47 @@
+// Type definitions for suman 1.1
```

Comment:
It doesn't look like you changed anything about the template besides adding the package name?

Reply:
I will do anything to get this to work, frankly I am lost, need help.

Reply:
It looks like suman provides its own typings in `lib/index.d.ts`. You shouldn't need to add these to DefinitelyTyped when they are already bundled with the package.